_includes/nav-links.html

```diff
@@ -16,3 +16,5 @@
   href="#shop">Shop</a>
 <a class="navbar-item about {% if page.url contains 'about' %} active-tab {% endif %}" id="about"
   href="#about">About</a> -->
+<a class="navbar-item code {% if page.url contains 'code' %} active-tab {% endif %}" id="code"
```

Comment:
Just a simple suggestion. We can use `code-of-conduct` here to avoid any clash or inconsistency with URLs with `code*` in the future.